nixos/modules/services/networking/pihole-ftl.md

```diff
@@ -0,0 +1,54 @@
+# Pi-hole {#module-services-networking-pihole-ftl}
+
+*Upstream documentation*: <https://docs.pi-hole.net/>
+
+[Pi-hole](https://pi-hole.net/) is a suite for creating and managing a local DNS
```

Comment:
Please break lines on sentence boundaries. Arbitrary breaks make review suggestion diffs needlessly messy.

Reply:
Sure, I've fixed this, thanks